mailbox/elasticsearch/src/main/java/org/apache/james/mailbox/elasticsearch/json/IndexableMessage.java

```diff
@@ -98,6 +101,17 @@ private static ZonedDateTime getSanitizedInternalDate(MailboxMessage message, Zo
             zoneId);
     }
 
+    private void generateText() {
+        this.text = Stream.of(from.serialize(),
```

Comment:
I don't think, as
* the only valid way to instanciate an IndexableMessage is to call the static method `from`
* joining large strings may have a cost
* there are no setter on the IndexableMessage attributes